media/css/pebbles/components/_buttons-download.scss

```diff
@@ -93,13 +95,72 @@ ul.download-list {
 // Product download buttons
 .download-button {
     .ios-download,
+    .windows-xpvista-download,
     .linux-arm-download,
     .unrecognized-download,
     .unsupported-download,
     .unsupported-download-osx,
     .nojs-download {
         display: none;
     }
+
+    .windows-xpvista-download {
```

Comment:
While this warning is certainly eye-catching, I do wonder if it's maybe a little excessive. I'd love to get @dzingeek's thoughts here from a design perspective. It doesn't look or feel like any other kind of warning we have on the site currently.

<img width="1347" alt="screen shot 2018-08-30 at 10 21 23" src="https://user-images.githubusercontent.com/400117/44842892-e97c7e00-ac3e-11e8-94f4-8ce2b1308529.png">

Reply:
Forgot to mention but this design is supposed to be tentative. The Protocol would probably have a style for such a warning?

Reply:
Right good call Alex. Let me think a bit more on this and also bring @vincejoyiv into the mix, he might already have some ideas on this.

Reply:
The warning above the button has to be prominent anyway IMO, because people are downloading the unsupported/insecure version. We intentionally allow to do it only for XP/Vista users.